src/PsrRequestFactoryInterface.php

```diff
@@ -5,9 +5,10 @@
 namespace Chubbyphp\WorkermanRequestHandler;
 
 use Psr\Http\Message\ServerRequestInterface;
+use Workerman\Connection\TcpConnection as WorkermanTcpConnection;
 use Workerman\Protocols\Http\Request as WorkermanRequest;
 
 interface PsrRequestFactoryInterface
 {
-    public function create(WorkermanRequest $workermanRequest): ServerRequestInterface;
+    public function create(WorkermanRequest $workermanRequest, WorkermanTcpConnection $workermanTcpConnection): ServerRequestInterface;
```

Comment:
If there is no way around a bc break i would suggest the following change:

```suggestion
    public function create(WorkermanTcpConnection $workermanTcpConnection, WorkermanRequest $workermanRequest): ServerRequestInterface;
```